Makefile

```
@@ -161,6 +161,9 @@ fmt: ## Check formatting of go sources.
 yamllint:
 	yamllint -c .yamllint deploy/examples/ --no-warnings
 
+.PHONY: lint
+lint: yamllint pylint shellcheck vet ## Run various linters
+
 .PHONY: pylint
 pylint:
 	pylint $(shell find $(ROOT_DIR)  -name '*.py') -E
```

While we are here, it might be a good time to fix this minor space/typo issue

```suggestion
	pylint $(shell find $(ROOT_DIR) -name '*.py') -E
```